motor_monitor_and_vibration/README.md

```diff
@@ -0,0 +1,186 @@
+
+## Hardware Initial Setup
+
+This initial setup configures the Notecard to provide activity and vibration monitoring.
+
+1. Assemble the Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/).
+2. Connect the Notecarrier to your PC with the micro USB cable.
+3. (Keep the Notecard connected to the computer with the in-browser terminal active until you have completed [Notecard Setup](#notecard-setup).)
+
+## Cloud Setup
+
+Sign up for a free account on [Notehub.io](https://notehub.io) and create a new project.
+
+## Notecard Setup
+
+In these steps, you will use the in-browser terminal to configure your Notecard to report the motor state and motion events to your Notehub project. You may also skip these individual steps and use the [#Notecard-CLI] to configure the Notecard using a single command.
+
+### Firmware
+
+The Notecard should use [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+### Configure Notehub Project and Connection Mode
+
+Set the ProductUID for the Notecard by pasting the command below into the in-browser terminal. Make sure to replace `com.your-company:your-product-name` with the ProductUID from your Notehub project, which can be found below your project's name in the dashboard at https://notehub.io. Also, replace the placeholder serial number (`"sn:" "motor-location"`) with the location of your fridge (e.g. "kitchen").
+
+```json
+{ "req": "hub.set", "mode": "periodic", "outbound": 5, "product": "com.your-company:your-product-name", "sn": "motor-location" }
+```
+
+This app uses `periodic` mode to minimize power use so that the solution can remain online for longer when mains power fails. `"outbound": 5` is the max wait time, in minutes, to sync outbound data from the Notecard. For more details, see [our Essential Requests documentation for `hub.set`](https://dev.blues.io/notecard/notecard-walkthrough/essential-requests/#notehub-configuration).
+
+### Configure Vibration Sensing
+
+Vibration sensing is configured using a number of distinct requests. Each request given below should be pasted into the in-browser terminal separately.
+
+#### `card.motion.mode`
+
+The [`card.motion.mode` requst](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-mode) enables motion sensing using the Notecard's built-in accelerometer. Motion is used to deterrmine how much the motor is vibration. The configuration below uses maximum sensitivity (`5`) and a 5 second period for each bucket of movement count.
```

Comment:
requst -> request and maybe move the "request" part out of the hyperlink. It's not included on the others below.

deterrmine -> determine

"Motion is used to deterrmine how much the motor is vibration" think this sentence needs reworking.

Reply:
I need a spell checker in VSCode :-)

Reply:
vibration -> vibrating. 

This plugin works well. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker